README.md

```diff
@@ -1,16 +1,9 @@
 # Forklift ETL
 
----
-
-## THIS TOOL IS NO LONGER SUPORTED.
-It has been succeded by [Empujar](https://github.com/taskrabbit/empujar)
```

Comment:
If you are removing this method, that assumes you want to support it @rgarver... is that the case?!